app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,30 @@
+class CommentsController < ApplicationController
+  before_action :load_chapter
+  before_action :require_login, only: :create
+
+  def create
+    @comment = @chapter.comments.build comment_params
+    @comment.user_id = current_user.id
+
+    if @comment.save
+      flash[:success] = t "commented"
+      redirect_to comic_chapter_url(@chapter.comic, @chapter)
```

Comment:
```suggestion
      redirect_to comic_chapter_url @chapter.comic, @chapter
```
nếu bỏ ra mà die rubocop thì thôi nhé anh :D

Reply:
ok nak chú